src/components/Auth/Auth.module.scss

```diff
@@ -0,0 +1,150 @@
+@import '../../styles/variables';
+
+div {
```

Comment:
This should be a `styles.tsx` if I'm not mistaken.  We've switched from using scss to makestyles/jss.

Reply:
Once you do that also update the colors use variables. There are a bunch already defined in the theme and if you want to add more they can be added there